lib/cmr/dmmt_client.rb

```diff
@@ -1,15 +1,32 @@
 module Cmr
   class DmmtClient < BaseClient
-    def dmmt_get_approved_proposals(params, token, request)
+    def dmmt_get_approved_proposals(token, request)
```

Comment:
if `request` is only needed for `test`, then we should give it a default value and try to only pass it when in `test`